src/util/xml.cpp

```diff
@@ -69,15 +69,22 @@ QDomElement XmlParse::openXMLFile(const QString& path, const QString& name) {
         qDebug() << "Could not open xml file:" << file.fileName();
         return QDomElement();
     }
+
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
+    const auto parseResult = doc.setContent(&file);
+    if (!parseResult) {
+        QString errorString = QString("%1 at line %2, column %3")
+                                      .arg(parseResult.errorMessage)
+                                      .arg(parseResult.errorLine)
+                                      .arg(parseResult.errorColumn);
```

Comment:
```suggestion
        QString errorString = QStringLiteral("%1 at line %2, column %3")
                                      .arg(parseResult.errorMessage, parseResult.errorLine, parseResult.errorColumn);
```
same below please.

Reply:
That was my initial attempt, but it doesn't compile because there are too many `arg` overloads and the compiler wants to interpret the last one as a `QChar` and fails.

Reply:
Ah, right. I forgot that the variadic overload only takes things that convert implicitly. In that case consider wrapping them in `QString::number`. I think thats more efficient than the longer temporary strings. Also `QStringLiteral` is nice.

Reply:
But the reason against multi-`arg` is that this causes a string alloc for each invocation. `QString::number` also causes a string alloc. So wouldn't we replace 3 allocations with 4?